lib/all_unique.js

```diff
@@ -1,5 +1,16 @@
 module.exports = {
   all_unique: function(str){
-    // Your code goes here.
+    	// If a string contains all unique characters, then it returns true
+	var sorted_str = str.split("").sort(); //turned the string into a sorted array of letters. any duplicate letters will be next to each each
+	var unique_letters = []; //This empty array will contain the unique letters
```

Comment:
Awesome work! As an optional challenge: How might you approach this _without_ using a container Array to collect unique characters? (The fact that you chose to sort the Array will be helpful)